lib/Jenkins/API.pm

```diff
@@ -24,15 +25,20 @@ has '_client' => (
         $client->setHost($self->base_url);
         if (defined($self->api_key) and defined($self->api_pass)) {
             $client->addHeader("Authorization", "Basic " .
-                               encode_base64($self->api_key . ':' . $self->api_pass)); 
+                               encode_base64($self->api_key . ':' . $self->api_pass));
         }
         return $client;
+    },
+    handles => {
+      response_code    => 'responseCode',
+      response_content => 'responseContent',
+      response_header  => 'responseHeader'
```

Comment:
I can't say I was aware the Moo did this.  That's neat.

Reply:
Yes please feel free to add me to the list of contributors.